include/libopencm3/stm32/common/usart_common_v2.h

```diff
@@ -194,6 +194,21 @@
 #define UART5_TDR			USART_TDR(UART5_BASE)
 #endif
 
+/** USART prescaler register (USART_PRESC) */
+#define USART_PRESC(usart_base)		MMIO32((usart_base) + 0x2C)
```

Comment:
which devices have this? is this really an extension? this doesn't seem right to include on things like f0 that don't have this?